persistence/persistence/src/main/java/one/microstream/persistence/exceptions/PersistenceExceptionStorerDeactivated.java

```diff
@@ -0,0 +1,38 @@
+package one.microstream.persistence.exceptions;
+
+/*-
+ * #%L
+ * MicroStream Persistence
+ * %%
+ * Copyright (C) 2019 - 2022 MicroStream Software
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import one.microstream.util.UtilStackTrace;
+
+/**
+ * This exception indicates that the storage is currently in a read only mode
+ * that denies the usage of {@link one.microstream.persistence.types.PersistenceStorer} methods.
+ * 
+ */
+@SuppressWarnings("serial")
+public class PersistenceExceptionStorerDeactivated extends PersistenceException
+{
+	@Override
+	public String getMessage()
```

Comment:
Not sure if this produces the right output, getThrowingMethodName returns the first method name of the stack, meaning the method in the user code which has called a store method.

Reply:
Is’s working as intended, the exeption is always thrown from a method in the ‘PersistenceStorerDeactivated‘ class.  The zeroth element of the stack trace is the method where throwable was created and thrown. Nevertheless I’ll fix as potential Index out of bounds error as the VM is allowed to return a zero sized stack trace if the VM has no stack strace.